src/test/java/org/openrewrite/java/spring/framework/MigrateResponseStatusExceptionTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.spring.framework;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.Issue;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class MigrateResponseStatusExceptionTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipeFromResources("org.openrewrite.java.spring.framework.MigrateResponseStatusException")
+          .parser(JavaParser.fromJavaVersion().classpathFromResources(new InMemoryExecutionContext(), "spring-core-5.3", "spring-beans-5.3", "spring-web-5.3"));
+    }
+
+    @DocumentExample
+    @Issue("https://github.com/openrewrite/rewrite-spring/issues/554")
+    @Test
+    void migrateResponseStatusExceptionGetStatusMethod() {
+        rewriteRun(
+          //language=java
+          java(
+            """
+              import org.springframework.http.HttpStatus;
+              import org.springframework.web.server.ResponseStatusException;
+              class A {
+                  void foo(ResponseStatusException e) {
+                      HttpStatus i = e.getStatus();
+                  }
+              }
+              """,
+            """
+              import org.springframework.http.HttpStatus;
+              import org.springframework.web.server.ResponseStatusException;
+              class A {
+                  void foo(ResponseStatusException e) {
+                      HttpStatus i = e.getStatusCode();
```

Comment:
I think this ought to be `HttpStatusCode` instead of `HttpStatus`, as per:
https://docs.spring.io/spring-framework/docs/6.0.0/javadoc-api/org/springframework/web/ErrorResponseException.html#getStatusCode()

Reply:
@Laurens-W is that something you can work in based on your earlier work on #589 ?

Reply:
Merging this iteration already as it's an improvement already; we'd need to change the test in a follow up PR to expect the variable and import change, and create a new dedicated recipe to handle the `ResponseStatusException getStatus()`.